pyproject.toml

```diff
@@ -1,3 +1,41 @@
+[build-system]
+requires = ["flit_core >=3.2,<4"]
+build-backend = "flit_core.buildapi"
+
+[project]
+name = "cohortextractor"
+description = ""
+version = "2.0.0"
+readme = "README.md"
+authors = [{name = "OpenSAFELY", email = "tech@opensafely.org"}]
+license = {file = "LICENSE"}
+classifiers = ["License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+)"]
+requires-python = ">=3.9"
+dependencies = [
```

Comment:
I think we should pin these here, if it's the only place we're going to define them.

Or else, figure out how to use these deps as the input to a `requirements.prod.in` that feeds into a `requirements.prod.txt`

Otherwise, we are not pinning the main deps, and always taking the latest versions, which is going to break things in CI.

Reply:
Yep good point, I'll take the current versions from the old requirements.prod.txt and we can start there.